inflammation/models.py

```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```

Comment:
Function name s_dev is not the best or self-explanatory. Please rename it to standard_deviation.